File: exemplos/JavaScript/nano/nano12.js
function main(): void = {
    var n: integer;
    var m: integer;
    var x: integer;
    
    n = 1;
    m = 2;
    x = 5;
    while (x > n) {
        if (n === m) {
            console.log(n);
        } else {
            console.log(0);
        }
    }
}

main();

File: exemplos/Lua/nano/nano12.lua
function main(): nil
	 local n, m, x:integer
	 n = 1
	 m = 2
	 x = 5
	 while x > n do
	       if n == m then
	       	  print(n)
	       else
	          print(0)
	       end
	       x = x - 1
	 end
end

main()
